### DISCUSSION TITLE
¿Cómo cambiar de versión de Jupyter dentro de Visual Studio Code?

### DISCUSSION THREAD

**Discussion by NoraBecerra** [LaunchX-InnovaccionVirtual/CursoIntroPython#233] [Q&A] [answered]
Hola buenas noches, he tenido problemas con la instalación de la extención de Jupyter que comentan que se puede instalar del año 2021 y solo aparece con el año 2022, podrian apoyarme por favor para hacer ese cambio de versión? Muchas gracias de antemano.

**Comment by EmmanuelMontes** [comment, marked as answer]
https://www.vsixhub.com/vsix/77766/

**Reply by NoraBecerra** [reply]
Muchas gracias por responder, excelente semana.

**Reply by EmmanuelMontes** [reply]
Igualmente Nora, a tus órdenes, saludos!
